migrations/20190623214136_defenses.js

```diff
@@ -42,6 +42,8 @@ exports.up = function(knex, Promise) {
         [nullability]()
```

Comment:
São nuláveis? Tem default?

Verifique isso. Corrigi a mesma coisa sua na meta 13.

Reply:
A intenção é ser nulável e não ter default. Confesso que não sei se isso está especificado em algum lugar, procurei no drive mas não achei.

Reply:
Agora que percebi aonde apontei. 

Mencionava as linhas 45 e 46

Reply:
Se for nulável, já coloca